plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -43,6 +46,7 @@ require('@sentry/tracing')
 
 // WARNING: Do not change this - it will essentially reset the consumer
 const KAFKA_CONSUMER_GROUP_ID = 'session-recordings-blob'
+const KAFKA_CONSUMER_GROUP_ID_OVERFLOW = 'session-recordings-blob-overflow'
```

Comment:
I wish github would let you comment on unchanged lines

reading through this again got me thinking of the various prometheus metrics. we have on our dashboard

I guess we need to change the dashboard so I can view metrics from not-overflow and overflow deployments separately rather than labelling them all on whether they're overflow

Reply:
All prom metrics inherit the `role` tag from the pod label, I'm planning on adding that tag as a dashboard var, keeping the default to the main consumer. We'll have to duplicate some kafka widgets, but the rest will be reusable without duping the whole dash.